src/components/DropMenu/DropMenu.jsx

```diff
@@ -504,6 +513,7 @@ const DropMenu = createClass({
 				'&-direction-up': isExpanded && direction === 'up',
 			}, className)} style={style}>
 				<ContextMenu
+					{...contextMenuProps}
```

Comment:
shouldn't this go after some stuff, since presumably you're trying to override them if you're passing them directly to a child component?